### ISSUE TITLE
Prop_physics on initial spawn can not be frozen (fixed)

### ISSUE THREAD

**Issue by Nick12506** [TheZoc/sourceforts-bug-tracker#72]
I can provide a example though title says it all.

**Comment by Nick12506** [comment]
Workaround is spawning the prop_physics in nofreeze zones, in most cases, the flag D would work.

**Comment by Nick12506** [comment]
Uh.. You can instantly unfreeze prop_physics props...

**Comment by TheZoc** [comment]
This bug report is unintelligible.

Need detailed description of what's going on and how to reproduce on regular games.

**Comment by Nick12506** [comment]
Okay, so the map includes a model being spawned in using prop_physics, the one inside the trigger zone was spawned in a nofreeze trigger.

The one outside the zone can not be frozen until it enters and exits the zone.

The one inside the zone can be frozen once removed from the zone. Both are being spawned in with prop_physics.

[sf_bugforzoc_propphysics.zip](https://github.com/TheZoc/sourceforts-bug-tracker/files/2701436/sf_bugforzoc_propphysics.zip)




After messing around with it on a live server, I noticed that those prop_physics models that we were able to freeze using the nofreeze method could be unfrozen by the enemy team on during combat without the unfreeze timer.


VMF of map \/

[sf_bugforzoc_propphysics.zip](https://github.com/TheZoc/sourceforts-bug-tracker/files/2701455/sf_bugforzoc_propphysics.zip)
